GoodreadsEDA/all_bks.py

```diff
@@ -21,52 +21,35 @@ def get_mv_release(bk_m):
                                        WHERE book_title="{bk_m}"
                                        ;""", conn)
         return df_rel["release_date"][0]
-    except IndexError as e:
-        print(f"{e} \n")
+    except IndexError:
+        traceback.print_exc()
         print("""Hmm... it looks as though the program cannot find the
 Release_Date table... ... did you run imdb_release.py yet?""")
 
 
 def create_even_samples(df, df2):
-    """Ballances the data so that the before and after values have an equal #
-    of sample sies."""
+    """Balances the data so that the before and after values have an equal #
+    of sample sites."""
 
     sample_diff = df.shape[0] - df2.shape[0]
     if sample_diff > 0:
-        df = df.sample(df2.shape[0])
-        return (df, df2)
+        return df.sample(df2.shape[0]), df2
     elif sample_diff < 0:
-        df2 = df2.sample(df.shape[0])
-        return (df, df2)
-    else:
-        return (df, df2)
+        return df, df2.sample(df.shape[0])
+    return df, df2
 
 
 # Create empty df to append each book to after preprocessing.
 df_all = pd.DataFrame()
 
 # Go through every title in the list above.
-for i, titl in enumerate(movie_titl):
```

Comment:
`enumerate` is really only necessary when you need the index and the value, but since you mostly just use the index to get the value, you can use this simpler syntax

Reply:
Yeah, I think this is an artifact of doing a poor job cleaning this script up before posting over to slack. That's for sure something I should have caught.